### DISCUSSION TITLE
Help with Setting Up Transport Equation for Advected Scalar

### DISCUSSION THREAD

**Discussion by JhonCordova** [AMReX-Combustion/PeleC#815] [Q&A]
Hello,

I need assistance with setting up the transport equation for an advected quantity in PeleC. I have included the directive `PELEC_NUM_ADV = 1` in the GNUmakefile. Subsequently, in the prob.H file, I initialized the values of this advected quantity within a sphere.

My objective is to maintain this quantity constant over time. To achieve this, I intend to introduce a source term that counteracts the advection term. I am using a specific [case](https://github.com/baperry2/PeleC/blob/54c8253eb53e887d45cf436d26dbb6d549d8de53/Exec/RegTests/EB-Vane/prob.H#L125) as a reference for adding the source term.

Could anyone provide guidance or suggestions on how to implement this effectively?

Thanks!

**Comment by baperry2** [comment]
I'm not sure if I understand what you're trying to do. Why set up an advected scalar, but then force it not to advect? You can add whatever you want in the custom source term function, so you could add a forcing that pushes the new state towards the old state, but I'm not sure if that's exactly your goal.
